### PR TITLE
Create Tooxic.json

### LINKED ISSUE #1431: Albin Håkanson
## Profile Screenshot

image

### PR DESCRIPTION
**Pull request by Tooxic**
resolves #1431 

## Checklist for profile addition

- [x] Name the PR `fix: add <Tooxic>`.
- [x] You've added file named `tooxic.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Tooxic.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (E-fais) [inline profiles/Tooxic.json:1]: LGTM

**Review by E-fais — APPROVED** [review]

**Review by E-fais — CHANGES_REQUESTED** [review]
change title of the pull request to;
fix: add Albin Håkanson .
Add a discription as follows:
resolves #1431

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**